vueui/src/components/view/records/list.vue

```diff
@@ -2,7 +2,12 @@
   <div>
     <vui-input-search style="display: block;" :input="records" v-model="filtered" :keys="['id', 'name', 'rank', 'sex', 'age', 'fingerprint', 'blood', 'dna']"/>
     <div v-for="record in filtered" :key="record.id">
-      <vui-button :params="{ setactive: record.id }" @click="state.activeview = state.defaultview" push-state>{{ record.id }}: {{ record.name }} ({{ record.rank }})</vui-button>
+      <div v-if="record.has_notes">
+        <vui-button :params="{ setactive: record.id }" icon="align-justify" @click="state.activeview = state.defaultview" push-state>{{ record.id }}: {{ record.name }} ({{ record.rank }})</vui-button>
+      </div>
+      <div v-else>
+        <vui-button :params="{ setactive: record.id }" @click="state.activeview = state.defaultview" push-state>{{ record.id }}: {{ record.name }} ({{ record.rank }})</vui-button>
+      </div>
```

Comment:
```suggestion
        <vui-button :params="{ setactive: record.id }" :icon="record.has_notes ? 'align-justify' : null" @click="state.activeview = state.defaultview" push-state>{{ record.id }}: {{ record.name }} ({{ record.rank }})</vui-button>
```
Not sure if this works, but it's better